src/app/footer/footer.component.html

```diff
@@ -18,6 +18,18 @@
                     <a>Privacy</a>
                 </div>
             </div>
+            <div class="row">
+                Tag Name: {{tag_name}}
+            </div>
+            <div class="row">
+                Id: {{id}}
+            </div>
+            <div class="row">
+               Created_at: {{created_at}}
+            </div>
+            <div class="row">
+                Published_at: {{published_at}}
+            </div>
         </div>
         <div class="col text-right"><a>Build:Worktrolly 2.3.0</a></div>
```

Comment:
The UI is not good only change 2.3.0 to tag_name

Reply:
its done